src/data/firefox-code-coverage-frontend.yaml

```diff
@@ -0,0 +1,36 @@
+name: firefox-code-coverage-frontend
+summary: Code-coverage-frontend allows determining code coverage changes for added lines per changeset.
+introduction: |
+  ## About firefox-code-coverage-frontend
+ 
+This project is the code coverage changeset viewer for Firefox. You can view the frontend running in [here](https://firefox-code-coverage.herokuapp.com/).
```

Comment:
Could you make this line & the next 2 lines align to the header `##About firefox-code-coverage-frontend` ? As you see now the preview build fails https://deploy-preview-236--codetribute.netlify.com/ and it can be fixed by aligning these three lines 😃